gcp/modules/gpii-istio/network_policy.tf

```diff
@@ -0,0 +1,14 @@
+resource "kubernetes_network_policy" "deny-default" {
```

Comment:
`kubernetes_network_policy` is not an Istio primitive, but I guess there is no better place for it to live atm...

Reply:
Yep I agree, I also don't like the name, perhaps the module would be better-named smth. like `gpii-namespace`, but I didn't want to rename the module, as it's not a good story with Terraform/terragrunt and creating one just for the policy also didn't seem to help things. I think logically it belongs with the namespace itself, just the module name is unfortunate.